MPSGameLift/Code/Source/MPSGameLiftServerSystemComponent.cpp

```diff
@@ -84,6 +95,8 @@ namespace MPSGameLift
                 "MPSGameLiftServerSystemComponent",
                 "Session requested by Amazon GameLift. Make sure to load into a multiplayer level before players join.");
         }
+
+        m_gamesessionNoPlayerShutdown.Enqueue(AZ::SecondsToTimeMs(sv_gamesessionNoPlayerShutdownTimeoutSeconds));
```

Comment:
nit: Add comment that this is cleared by the connection handler when a player connects (ie see MPSGameLiftServerSystemComponent::m_connectionAquiredEventHandler 

as its not clear when this would be cleared.

Reply:
fixed